packages/kbn-openapi-bundler/src/__test__/reduce_all_of/merge_all_of_items_with_inlined_refs_in_different_document_branches/spec.schema.yaml

```
@@ -0,0 +1,56 @@
+openapi: 3.0.3
+info:
+  title: Test endpoint GET
+  version: '2023-10-31'
+paths:
+  /api/some_api:
+    get:
+      operationId: TestEndpointGet
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  responseSchemaFieldA:
+                    $ref: '#/components/schemas/SchemaToInline'
+                  responseSchemaFieldB:
+                    $ref: '#/components/schemas/MySchema'
+
+components:
+  schemas:
+    MySchema:
+      type: object
+      properties:
+        allOf:
+          - $ref: '#/components/schemas/SchemaToInline'
+          - type: object
+            properties:
+              mySchemaSubfield:
+                type: boolean
```

This is also something invalid. Did you mean:

```suggestion
    MySchema:
      allOf:
        - $ref: '#/components/schemas/SchemaToInline'
        - type: object
          properties:
            mySchemaSubfield:
              type: boolean
```